packages/cli/src/util/dev/server.ts

```diff
@@ -689,17 +690,39 @@ export default class DevServer {
     }
 
     // legacy NOW_REGION env variable
-    runEnv['NOW_REGION'] = 'dev1';
     buildEnv['NOW_REGION'] = 'dev1';
```

Comment:
@styfle should I just remove this entirely?

Reply:
No, I think we need to keep it for backwards compat

Reply:
You can move it to the end after constructing `allEnv` though